test/suites/container_devices_nic_routed.sh

```diff
@@ -66,6 +66,20 @@ test_container_devices_nic_routed() {
     mtu=1600
   lxc start "${ctName}"
 
+  # Container has started.
+  # It should no longer be possible to add routed nic devices with the same IP addresses.
+  lxc init testimage dummy
```

Comment:
You're not deleting this instance after the test, that's likely the reason for the test failures.

Reply:
Ah I didn't realise that making a change to one test might cause other tests to fail. I've now deleted the dummy instance and all the tests are passing locally.